doc/shared/sunadaptcontroller/SUNAdaptController_Description.rst

```diff
@@ -302,6 +346,27 @@ note these requirements below. Additionally, we note the behavior of the base SU
 
       retval = SUNAdaptController_UpdateH(C, h, dsm);
 
+.. c:function:: SUNErrCode SUNAdaptController_UpdateMRITol(SUNAdaptController C, sunrealtype H, sunrealtype tolfac, sunrealtype DSM, sunrealtype dsm)
+
+   Notifies a controller of type ``SUN_ADAPTCONTROLLER_MRI_TOL`` that a successful time step
+   was taken with slow stepsize *H* and fast relative tolerance factor *tolfac*, and that the
+   step had slow and fast local error factors *DSM* and *dsm*, indicating that these can be
+   saved for subsequent controller functions. This is typically relevant for controllers that
+   store a history of either step sizes or error estimates for performing the estimation process.
+
+   :param C:  the :c:type:`SUNAdaptController` object.
+   :param H:  the successful slow step size.
+   :param tolfac:  the successful fast time scale relative tolerance factor.
+   :param DSM:  the successful slow temporal error estimate.
+   :param dsm:  the successful fast temporal error estimate.
+   :return: :c:type:`SUNErrCode` indicating success or failure.
+
+   Usage:
```

Comment:
This usage block does not add any value as-is IMO. I would suggest removing it or updating it with more context.